src/l500/l500-color.cpp

```diff
@@ -358,6 +368,33 @@ namespace librealsense
         if (_state != sensor_state::OWNED_BY_USER)
             throw wrong_api_call_sequence_exception("tried to start an unopened sensor");
 
+#ifdef __ANDROID__
+        if (supports_option(RS2_OPTION_ENABLE_WEAK_USB_HOST_WA))
+        {
+            auto usb_perf_enabled = get_option(RS2_OPTION_ENABLE_WEAK_USB_HOST_WA).query();
+
+            if (usb_perf_enabled)
+            {
+                // TPROC USB Granularity and TRB threshold settings for improved performance and stability
+                // on hosts with weak cpu and system performance
+                // settings values are validated through many experiments, do not change unless
+                try {
+                    // endpoint 5 - 32KB
+                    command cmdTprocGranEp5(ivcam2::TPROC_USB_GRAN_SET, 5, 32);
+                    _owner->_hw_monitor->send(cmdTprocGranEp5);
+
+                    command cmdTprocThresholdEp5(ivcam2::TPROC_TRB_THRSLD_SET, 5, 1);
+                    _owner->_hw_monitor->send(cmdTprocThresholdEp5);
+
+                    LOG_INFO("Color usb tproc granularity and TRB threshold updated.");
```

Comment:
Should be debug -- no need to see it in the Viewer.

Reply:
fixed